lib/web/ui/access_request.go

```diff
@@ -0,0 +1,170 @@
+package ui
+
+import (
+	"time"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/types"
+)
+
+// AccessRequest describes a request's current state.
```

Comment:
Why do we need this stuff here?

Reply:
This was originally needed for some assistant code to get the struct definitions the web frontend used. We did refactor this so the move from `e` wasn't needed after all. Removed.